src/components/AddForm.js

```diff
@@ -6,28 +6,41 @@ import {
   addTripsToLocalStorage,
   getTripsFromLocalStorage,
 } from "../services/tripsStorage";
+import axios from "axios";
 
 function AddForm() {
   const [destinationInput, setDestinationInput] = useState("");
   const [activitiesInput, setActivitiesInput] = useState("");
   const [restaurantsInput, setRestaurantsInput] = useState("");
   const [notesInput, setNotesInput] = useState("");
+  const [imageUpload, setImageUpload] = useState("");
 
   function handleSubmit(e) {
     e.preventDefault();
-    addTripsToLocalStorage({
-      id: `${destinationInput}`,
-      destination: destinationInput,
-      activities: activitiesInput,
-      restaurants: restaurantsInput,
-      notes: notesInput,
-    });
-    const trips = getTripsFromLocalStorage();
-    console.log(trips);
-    setDestinationInput("");
-    setActivitiesInput("");
-    setRestaurantsInput("");
-    setNotesInput("");
+    const formData = new FormData();
+    formData.append("file", imageUpload);
+    formData.append("upload_preset", "tyikvr8a");
+
+    axios
```

Comment:
You don't need to use axios here. You could just use `fetch` and uninstall `axios`
You can take a look how to use `fetch` to make a post request here:

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#supplying_request_options

Reply:
Also Johan did already a post request with axios in his project, so you can ask him as well :)